### PR TITLE
Adding new release notes entry

### PR DESCRIPTION
**Pull request by RobertAnderson-okta**
## Description:
- **What's changed?** Monthly release notes entry
- **Is this PR related to a Monolith release?** Yes 2020.05.0

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2020-05-0/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+| [Rate limit headers no longer returned on cached static endpoints](#rate-limit-headers-no-longer-returned-on-cached-static-endpoints)                               | May 6, 2020              |
+| [OAuth for Okta enabled for Trusted Origins, Sessions, and Custom Templates APIs](#oauth-for-okta-enabled-for-trusted-origins-sessions-and-custom-templates-apis)   | May 6, 2020              |
+| [Updated behavior for logging of invalid use by OAuth 2.0 Client](#updated-behavior-for-logging-of-invalid-use-by-oauth-2-0-client)                                 | May 6, 2020              |
+| [Bugs fixed in 2020.05.0](#bugs-fixed-in-2020-05-0)                                                                                                                 | May 6, 2020              |
+
+### Logging of successful password import
+
+A Syslog Event is now generated with details about the success or failure of the password import attempt when a user with an imported password has successfully logged in to Okta. <!-- OKTA-283126 -->
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-05-0/index.md:17]:
```suggestion
A System Log Event is now generated with details about the success or failure of the password import attempt when a user with an imported password has successfully signed in to Okta. <!-- OKTA-283126 -->
```

File: packages/@okta/vuepress-site/docs/release-notes/2020-05-0/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+* `/.well-known/openid-configuration`
+* `/.well-known/oauth-authorization-server`
+* `/oauth2/{authorizationServerId}/v1/keys`
+* `/oauth2/{authorizationServerId}/.well-known/openid-configuration`
+* `/oauth2/{authorizationServerId}/.well-known/oauth-authorization-server`
+<!-- OKTA-289849 -->
+
+### OAuth for Okta enabled for Trusted Origins, Sessions, and Custom Templates APIs
+
+OAuth for Okta is now enabled for the Trusted Origins API, the Sessions API, and the Custom Templates API. See [Scopes & supported endpoints](/docs/guides/implement-oauth-for-okta/scopes/). <!-- OKTA-286819 -->
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-05-0/index.md:33]: We should link to the respective API reference pages.

File: packages/@okta/vuepress-site/docs/release-notes/2020-05-0/index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+### Updated behavior for logging of invalid use by OAuth 2.0 Client
+
+The [previously announced logging behavior](https://developer.okta.com/docs/release-notes/2017-23/#additional-logging-for-invalid-use-by-oauth-2-0-client) has been updated. Invalid client_secret warnings are now triggered by 5 invalid attempts (consecutive or not) within a 24 hour period. <!-- OKTA-288030 -->
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-05-0/index.md:37]:
```suggestion
The [previously announced logging behavior](/docs/release-notes/2017-23/#additional-logging-for-invalid-use-by-oauth-2-0-client) has been updated. Invalid `client_secret` warnings are now triggered by 5 invalid attempts (consecutive or not) within a 24 hour period. <!-- OKTA-288030 -->
```

File: packages/@okta/vuepress-site/docs/release-notes/2020-05-0/index.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+### Bugs fixed in 2020.05.0
+
+* When logging in a federated user using the `/oauth/v1/authorize` endpoint with consent enabled and the prompt parameter set to login, the widget failed with an error. (OKTA-290760)
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-05-0/index.md:41]:
```suggestion
* When signing in a federated user using the `/oauth/v1/authorize` endpoint with consent enabled and the `prompt` parameter set to `login`, the Sign-In Widget failed with an error. (OKTA-290760)
```

File: packages/@okta/vuepress-site/docs/release-notes/2020-05-0/index.md (hunk 2 above)
Comment (RobertAnderson-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2020-05-0/index.md]: Thanks, just updated that.

**Pull request merged**